app/policies/work_policy.rb

```diff
@@ -8,7 +8,7 @@ def limit
   end
 
   def show?
-    (record.discover_access?(user) && published?) || edit?
+    published? && (record.discover_access?(user) || edit?)
```

Comment:
@rschenk If I'm reading this right, nobody (not even admins) can view unpublished _works_. Work _versions_ that are in draft state should still be fine, but I'm wondering if there's a need to redirect to the draft work version if an admin, or someone else, attempts to view the unpublished work uuid.

I may be over-thinking this, because in most cases, we aren't going to be passing around uuids for unpublished works, just for work versions since that's the url in the dashboard.

Reply:
You're reading that right. I'm not sure if this is the way we want to go, but it does seem that way. The resource page for a _Work_ requires that it have at least one published version. If there is not a published version it will panic and 500. 

We could either make the resource page gracefully handle unpublished works, or we disallow them entirely (what this does).